modules/clientnotify.cpp

```diff
@@ -22,14 +22,19 @@ using std::set;
 class CClientNotifyMod : public CModule {
   protected:
     CString m_sMethod;
+    CString m_sNewNotifyOn;
```

Comment:
This var is not needed, as you have 2 bools for it already

Reply:
I have now split upd the option `NewNotifyOn` into `NotifyOnNewIP` and `NotifyOnNewID`, so the string is removed!